.bazelci/presubmit.yml

```diff
@@ -440,16 +440,17 @@ tasks:
     test_targets:
       - "//..."
     build_flags: *aspects_flags
-  crate_universe_examples_macos:
-    name: Crate Universe Examples
-    platform: macos
-    working_directory: examples/crate_universe
-    run_targets: *crate_universe_vendor_example_targets
-    build_targets:
-      - "//..."
-    test_targets:
-      - "//..."
-    build_flags: *aspects_flags
+  # TODO: https://github.com/bazelbuild/rules_rust/issues/1814
+  # crate_universe_examples_macos:
```

Comment:
It'd be nice to delete the use of `rules_foreign_cc` from this job. Currently it's used to build openssl but it'd be nice to switch the use of openssl to rustls